### ISSUE TITLE
django bug about str not having method 'round'

### ISSUE THREAD

**Issue by hamilton** [mozilla/glam#267]
This is for `http_altsvc_entries_per_header`.

The error is not particularly useful, obviously, but take a look at the response payload on the bottom left.

![Screen Shot 2020-02-28 at 3 46 47 PM](https://user-images.githubusercontent.com/95735/75595527-ab356a00-5a41-11ea-95b0-52b6aced6b15.png)

**Comment by hamilton** [comment]
@spasovski I wonder if there's a way to more gracefully catch this class of errors. I am now realizing that this is probably an older bug since I've seen this exact message before over the past few months for some probes.

**Comment by robhudson** [comment]
This happens b/c there's a `"NaN"` in the JSON data from upstream which shouldn't be happening soon once the dupes and stuff are cleaned up.

Just in case, /cc @emtwo 

I could add some checks in the views to guard against this if we want it fixed sooner.

Example data:
```
[{'key': '5', 'value': 'NaN'}, {'key': '25', 'value': 'NaN'}, {'key': '50', 'value': 'NaN'}, {'key': '75', 'value': 'NaN'}, {'key': '95', 'value': 'NaN'}]
```

**Comment by hamilton** [comment]
This issue will be immediately solved w/ the new data import. Re: 500 errors, we have filed an issue for it (#283) and a fix (#282). Closing this one in favor of the others.
